projects/cm_mcu/LocalTasks.c

```diff
@@ -281,9 +291,10 @@ struct sm_command_t sm_command_ffldaq_f2[] = {
     {1, 0x00, 0x16, 2, "FF_TEMPERATURE", 0xff, "C", PM_STATUS},
     {1, 0x00, 0x03, 1, "FF_LOS_ALARM", 0xff, "", PM_STATUS},
     {1, 0x00, 0x05, 1, "FF_CDR_LOL_ALARM", 0xff, "", PM_STATUS},
-
+    {2, 0x00, 0x22, 1, "FF_OPT_POW", 0xff, "mw", PM_STATUS}, // read 4 Rx-ch registers with increasing addresses
```

Comment:
I was expecting to see an entry for every one of the four channels here, where are they?

Reply:
As I have mentioned, we store an average optical power per FF with 25Gbs. The magic of reading each channel is [here](https://github.com/apollo-lhc/cm_mcu/blob/b6b3539f78c9617dd99e43e54c2f3551ee39504e/projects/cm_mcu/MonitorI2CTask.c#L228-L232).

Reply:
do we have to use magic? I thought we would just enumerate the registers we want to read, no magic required.

Reply:
I understand your point that the registers can be explicitly added here. Perhaps you are suggesting that we should change the CLI command for reading optical power to shows this value in all channels? but for now we can only send an average value to zynq, right?

Reply:
Dan is going to change the data size so we can send the full information soon ... so we should be ready for that. I guess we can merge this PR with just the average value for sending.